github.go

```diff
@@ -61,6 +62,7 @@ func NewGithubClient(
 	token string,
 	releaseFilter ReleaseFilter,
 	assetFilter AssetFilter,
+	connectionTimeout time.Duration,
```

Comment:
It looks like this is never used

Reply:
Sorry, that was some IDE mess-up 🙈 
Fixed now 🙂